pkg/config/default.go

```diff
@@ -398,6 +399,12 @@ func defaultConfigFromMemory() (*EngineConfig, error) {
 
 	c.PodExitPolicy = defaultPodExitPolicy
 
+	dirname, err := os.UserHomeDir()
```

Comment:
This could break in a rootless context as the `os` package does not understand user namespaces.  I suggest using `containers/storage/pkg/homedir.Get()` instead.  That also works on non-Linux machines.

Reply:
ah! this is why the teats are breaking (I think) thanks for pointing that out